docs/core/diagnostics/dotnet-trace.md

```diff
@@ -86,6 +86,9 @@ dotnet-trace collect [--buffersize <size>] [--clreventlevel <clreventlevel>] [--
 
   Sets the size of the in-memory circular buffer, in megabytes. Default 256 MB.
 
+  > [!NOTE]
+  > If the target process is writing events too frequently, it can overflow this buffer which results in some events to get dropped. When too many events are getting dropped, increasing the buffer size may help.
```

Comment:
```suggestion
  > If the target process writes events too frequently, it can overflow this buffer and some events might be dropped. If too many events are getting dropped, increase the buffer size.
```

Reply:
Thanks for the feedback @gewarren - I had to reword it a little bit because the "may help" phrasing here was chosen deliberately because it doesn't always help. I've updated the PR to reflect the suggestions minus that part and made some clarifications to that.